src/util/Permissions.js

```diff
@@ -18,6 +18,17 @@ class Permissions extends BitField {
    * @typedef {string|number|Permissions|PermissionResolvable[]} PermissionResolvable
    */
 
+  /**
+   * Checks whether the bitfield has a permission, or any of multiple permissions.
+   * @param {PermissionResolvable} permission Permission(s) to check for
+   * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
+   * @returns {boolean}
+   */
+  any(permission, checkAdmin = true) {
+    if (checkAdmin && super.has(this.constructor.FLAGS.ADMINISTRATOR)) return true;
```

Comment:
it's kind of messy to split up the boolean logic like that such that you end up with an `if(true) return true; else blah;`
```suggestion
    return (checkAdmin && super.has(this.constructor.FLAGS.ADMINISTRATOR)) || super.any(permission);
```

Reply:
For consistency, I've pushed a new commit that makes this change to both `has` and `any`, not just this method.